WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java

```diff
@@ -241,9 +243,26 @@
     private WPSwitchPreference mJpUseTwoFactorPref;
 
     // Speed up settings
-    private PreferenceScreen mSpeedUpYourSiteSettings;
-    private WPSwitchPreference mServeImagesFromOurServers;
     private WPSwitchPreference mLazyLoadImages;
+    private WPSwitchPreference mLazyLoadImagesNested;
+
+    // Jetpack media settings
+    private WPSwitchPreference mAdFreeHosting;
```

Comment:
Should we rename this one to mAdFreeVideoHosting to be consistent with naming in JetpackSettingsModel?